libs/utils/env.py

```diff
@@ -474,19 +485,36 @@ def _init_target(self, force = False):
             # Brackets are there to let the output dir be created automatically
                                     '--workload-automation-vio={}'])
 
+            # Add variant suffix
+            suffix = os.path.splitext(os.path.basename(
+                     system_info['platform']['description']))[0]
+            board_file = self.conf['board'].lower() + suffix + '.json';
+            self._board_path = os.path.join(basepath, 'libs/utils/platforms',
+                                      board_file)
+            if not os.path.exists(self._board_path):
+                self._log.warning('Board descriptor file %s was not found.',
+                                  self._board_path)
+                self._board_path = None
+                board = None
+            else:
+                board = self._load_board()
```

Comment:
@ionela-voinescu That looks good to me, I agree it's more sensible to have this done just for gem5. Do you want me to add it to this PR ?